src/main/webapp/feedback.js

```diff
@@ -37,3 +37,9 @@ function getScheduledInterviewId() {
 function getRole() {
   return new URLSearchParams(window.location.search).get('role'); 
 }
+
+// Sends the feedback forms.
+function doSubmit() {
+  document.feedbackForm.interviewId.value = getScheduledInterviewId();
+  return true;
```

Comment:
Q: why does this return true if it's not being called anywhere?

Reply:
It is called when the form is submitted. I use this to include the scheduledInterview's id in the servlet request

Reply:
Ok l see it being called but the return value not being used.